### YamlMime:ManagedReference
items:
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  commentId: T:Xunit.SimpleRunner.ErrorMessageType
  id: ErrorMessageType
  parent: v3.3.0.0:Xunit.SimpleRunner
  children:
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.CatastrophicError
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestAssemblyCleanupFailure
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestCaseCleanupFailure
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestClassCleanupFailure
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestCleanupFailure
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestCollectionCleanupFailure
  - v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestMethodCleanupFailure
  langs:
  - csharp
  - vb
  name: ErrorMessageType
  nameWithType: ErrorMessageType
  fullName: Xunit.SimpleRunner.ErrorMessageType
  type: Enum
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: ErrorMessageType
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 5
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An enumeration which indicates the type of error message (for <xref href="v3.3.0.0:Xunit.SimpleRunner.AssemblyRunnerOptions.OnErrorMessage" data-throw-if-not-resolved="false"></xref>).
  example: []
  syntax:
    content: public enum ErrorMessageType
    content.vb: Public Enum ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.CatastrophicError
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.CatastrophicError
  id: CatastrophicError
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: CatastrophicError
  nameWithType: ErrorMessageType.CatastrophicError
  fullName: Xunit.SimpleRunner.ErrorMessageType.CatastrophicError
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: CatastrophicError
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 8
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception occurred that disrupted the execution engine
  example: []
  syntax:
    content: CatastrophicError = 1
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestAssemblyCleanupFailure
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.TestAssemblyCleanupFailure
  id: TestAssemblyCleanupFailure
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: TestAssemblyCleanupFailure
  nameWithType: ErrorMessageType.TestAssemblyCleanupFailure
  fullName: Xunit.SimpleRunner.ErrorMessageType.TestAssemblyCleanupFailure
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: TestAssemblyCleanupFailure
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 11
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception happened while cleaning up from the test assembly
  example: []
  syntax:
    content: TestAssemblyCleanupFailure = 10
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestCollectionCleanupFailure
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.TestCollectionCleanupFailure
  id: TestCollectionCleanupFailure
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: TestCollectionCleanupFailure
  nameWithType: ErrorMessageType.TestCollectionCleanupFailure
  fullName: Xunit.SimpleRunner.ErrorMessageType.TestCollectionCleanupFailure
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: TestCollectionCleanupFailure
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 14
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception happened while cleaning up from the test collection
  example: []
  syntax:
    content: TestCollectionCleanupFailure = 20
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestClassCleanupFailure
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.TestClassCleanupFailure
  id: TestClassCleanupFailure
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: TestClassCleanupFailure
  nameWithType: ErrorMessageType.TestClassCleanupFailure
  fullName: Xunit.SimpleRunner.ErrorMessageType.TestClassCleanupFailure
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: TestClassCleanupFailure
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 17
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception happened while cleaning up from the test class
  example: []
  syntax:
    content: TestClassCleanupFailure = 30
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestMethodCleanupFailure
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.TestMethodCleanupFailure
  id: TestMethodCleanupFailure
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: TestMethodCleanupFailure
  nameWithType: ErrorMessageType.TestMethodCleanupFailure
  fullName: Xunit.SimpleRunner.ErrorMessageType.TestMethodCleanupFailure
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: TestMethodCleanupFailure
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 20
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception happened while cleaning up from the test method
  example: []
  syntax:
    content: TestMethodCleanupFailure = 40
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestCaseCleanupFailure
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.TestCaseCleanupFailure
  id: TestCaseCleanupFailure
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: TestCaseCleanupFailure
  nameWithType: ErrorMessageType.TestCaseCleanupFailure
  fullName: Xunit.SimpleRunner.ErrorMessageType.TestCaseCleanupFailure
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: TestCaseCleanupFailure
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 23
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception happened while cleaning up from the test case
  example: []
  syntax:
    content: TestCaseCleanupFailure = 50
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType.TestCleanupFailure
  commentId: F:Xunit.SimpleRunner.ErrorMessageType.TestCleanupFailure
  id: TestCleanupFailure
  parent: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  langs:
  - csharp
  - vb
  name: TestCleanupFailure
  nameWithType: ErrorMessageType.TestCleanupFailure
  fullName: Xunit.SimpleRunner.ErrorMessageType.TestCleanupFailure
  type: Field
  source:
    remote:
      path: src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
      branch: d0213fc4e5d673d4c3abcb807d2eab72cfd80fe9
      repo: https://github.com/xunit/xunit
    id: TestCleanupFailure
    path: ../src/xunit.v3.runner.utility/SimpleRunner/ErrorMessageType.cs
    startLine: 26
  assemblies:
  - xunit.v3.runner.utility.netfx
  namespace: Xunit.SimpleRunner
  summary: An unhandled exception happened while cleaning up from the test
  example: []
  syntax:
    content: TestCleanupFailure = 60
    return:
      type: Xunit.SimpleRunner.ErrorMessageType
references:
- uid: v3.3.0.0:Xunit.SimpleRunner.AssemblyRunnerOptions.OnErrorMessage
  commentId: P:Xunit.SimpleRunner.AssemblyRunnerOptions.OnErrorMessage
  href: Xunit.SimpleRunner.AssemblyRunnerOptions.OnErrorMessage.html#Xunit_SimpleRunner_AssemblyRunnerOptions_OnErrorMessage
  name: OnErrorMessage
  nameWithType: AssemblyRunnerOptions.OnErrorMessage
  fullName: Xunit.SimpleRunner.AssemblyRunnerOptions.OnErrorMessage
- uid: v3.3.0.0:Xunit.SimpleRunner
  commentId: N:Xunit.SimpleRunner
  href: Xunit.html
  name: Xunit.SimpleRunner
  nameWithType: Xunit.SimpleRunner
  fullName: Xunit.SimpleRunner
  spec.csharp:
  - uid: v3.3.0.0:Xunit
    name: Xunit
    href: Xunit.html
  - name: .
  - uid: v3.3.0.0:Xunit.SimpleRunner
    name: SimpleRunner
    href: Xunit.SimpleRunner.html
  spec.vb:
  - uid: v3.3.0.0:Xunit
    name: Xunit
    href: Xunit.html
  - name: .
  - uid: v3.3.0.0:Xunit.SimpleRunner
    name: SimpleRunner
    href: Xunit.SimpleRunner.html
- uid: v3.3.0.0:Xunit.SimpleRunner.ErrorMessageType
  commentId: T:Xunit.SimpleRunner.ErrorMessageType
  parent: v3.3.0.0:Xunit.SimpleRunner
  href: Xunit.SimpleRunner.ErrorMessageType.html
  name: ErrorMessageType
  nameWithType: ErrorMessageType
  fullName: Xunit.SimpleRunner.ErrorMessageType
memberLayout: SeparatePages
